misc/dbt-materialize/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # dbt-materialize Changelog
 
-## Unreleased - 2022-10-16
+## Unreleased - 2022-10-28
+
+* Return the correct connection keys to the user on `dbt debug`.
+
+* Fail on index creation if the index already exists.
```

Comment:
```suggestion
```

Reply:
I don't think this needs a note, since it's expected behavior (IIRC, we also changed it for views before and didn't include it in the changelog).